Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This Dockerfile used only for the API tests.
 
-FROM golang:1.18
+FROM golang:1.18.5
```

Comment:
Do we want to specify exact patch versions? That means that we should update it more often.

Reply:
Good point, Updated to just 1.19